client/src/components/GraphViewerComponent/GraphViewerCytoscapeComponent/GraphViewerCytoscapeComponent.js

```diff
@@ -474,7 +485,25 @@ export class GraphViewerCytoscapeComponent extends React.Component {
     });
   }
 
+  saveLayout() {
+    storageService.saveGraphLayoutPositions(this.layout, this.query, sessionService.getCurrentGraphLayoutPositions(this.layout, this.query));
+    const msg = `${this.layout} layout saved`;
+    print(msg);
+    this.setState({ notificationMessageIsVisible: true, notificationMessage: msg });
+  }
+
+  clearLayout() {
+    sessionService.clearGraphLayout(this.layout, this.query);
+    storageService.clearGraphLayout(this.layout, this.query);
```

Comment:
Why are we storing the layout in two places?

Maybe we need to have a single service that wraps the interaction between these two services? I.e. a LayoutStorageService or similar?

Reply:
Created LayoutService class to handle the calls of each respective component in a unified class